src/main/java/com/google/error_analyzer/backend/StoreLogs.java

```diff
@@ -31,30 +33,34 @@
     private static final Logger logger = LogManager.getLogger(StoreLogs.class);
     private static final String LINE_BREAK = "\\r?\\n";
     private StoreLogHelper storeLogHelper = new StoreLogHelper();
-    private String ERROR_TEMPLATE_RESPONSE = "\t\t\t<h2> Could not store file %1$s</h2>";
-    public static final String FILE_STORED_RESPONSE =
-        "\t\t\t<h2> File Stored</h2>";
-    public static final String FILE_ALREADY_EXISTS_RESPONSE =
-        "\t\t\t<h2> Sorry! the file already exists. " +
-        "Please try with a different file name</h2>";
+    private String ERROR_TEMPLATE_RESPONSE =
+        "\t\t\t<h2> Could not store file %1$s</h2>";
+    public static final String FILE_STORED_TEMPLATE_RESPONSE =
+        "\t\t\t<h2> File %1$s Stored</h2>";
+    public static final String FILE_EMPTY_TEMPLATE_RESPONSE =
+        "\t\t\t<h2> Sorry! the file %1$s is empty</h2>";
     public DaoInterface logDao = new LogDao();
 
-    //Calls the method StoreLog if an index with name fileName does not 
-    //exist in db
-    public String checkAndStoreLog(String fileName, String log) {
+    //stores the logs into database with appropriate filename
+    public String checkAndStoreLog(HttpServletRequest request, String fileName,
+        String log) {
         try {
-            if (logDao.fileExists(fileName)) {
-                logger.error(String.format("File %s already exists", fileName));
-                return FILE_ALREADY_EXISTS_RESPONSE;
-            } else {
-                final String response = storeLog(fileName, log);
+            String indexName = LogDaoHelper.getIndexName(request, fileName);
+            indexName = getUniqueIndexName(indexName);
+            final String response = storeLog(indexName, log);
+            fileName = LogDaoHelper.getFileName(request, indexName);
+            if (response.equals(FILE_EMPTY_TEMPLATE_RESPONSE)) {
+                logger.info(String.format("File %s is empty", fileName));
+            }
+            else {
                 logger.info(String.format("File %s stored", fileName));
-                return response;
             }
+            return String.format(response, fileName);
         } catch (Exception e) {
             final String ERROR_RESPONSE =
                 String.format(ERROR_TEMPLATE_RESPONSE, e);
-            logger.error(String.format("Could not store file %1$s %2$s", fileName, e));
+            logger.error(String.format("Could not store file %1$s %2$s",
```

Comment:
logger.error(errorResponse);

Reply:
Unnecessary html tags will added to logger